docs/dev/query-type-conversion.md

```diff
@@ -147,15 +148,15 @@ public enum ExprCoreType implements ExprType {
   DOUBLE(FLOAT),
 
   STRING(UNDEFINED),
-  BOOLEAN(STRING), // PR: change STRING's parent to BOOLEAN
+  BOOLEAN(STRING), // #166 changes: STRING's parent to BOOLEAN
 
   /**
    * Date.
    */
-  TIMESTAMP(UNDEFINED),
-  DATE(UNDEFINED),
-  TIME(UNDEFINED),
-  DATETIME(UNDEFINED),
+  DATE(STRING),  // #171 changes: datetime types parent to STRING
```

Comment:
This seems like a mismatch from `docs/dev/img/type-hierarchy-tree-final.png`. Is this correct?

Reply:
I don't see a mismatch. Can you clarify please?

Reply:
Disregard this comment. I just misunderstood.